specs/lightclient/validator.md

```diff
@@ -0,0 +1,377 @@
+      - [`ContributionAndProof`](#contributionandproof)
+      - [`SignedContributionAndProof`](#signedcontributionandproof)
+- [Validator assignments](#validator-assignments)
+  - [Sync Committee](#sync-committee)
+  - [Lookahead](#lookahead)
+- [Beacon chain responsibilities](#beacon-chain-responsibilities)
+  - [Block proposal](#block-proposal)
+    - [Preparing a `BeaconBlock`](#preparing-a-beaconblock)
+    - [Constructing the `BeaconBlockBody`](#constructing-the-beaconblockbody)
+      - [Sync committee](#sync-committee)
+    - [Packaging into a `SignedBeaconBlock`](#packaging-into-a-signedbeaconblock)
+  - [Attesting and attestation aggregation](#attesting-and-attestation-aggregation)
+  - [Sync committees](#sync-committees)
+    - [Sync committee signatures](#sync-committee-signatures)
+      - [Prepare sync committee signature](#prepare-sync-committee-signature)
+      - [Broadcast sync committee signature](#broadcast-sync-committee-signature)
+    - [Sync committee contributions](#sync-committee-contributions)
+      - [Aggregation selection](#aggregation-selection)
+      - [Construct sync committee contribution](#construct-sync-committee-contribution)
+      - [Broadcast sync committee contribution](#broadcast-sync-committee-contribution)
+- [Sync committee subnet stability](#sync-committee-subnet-stability)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This document represents the expected behavior of an "honest validator" with respect to HF 1 of the Ethereum 2.0 protocol. It builds on the [previous document for the behavior of an "honest validator" from Phase 0](../phase0/validator.md) of the Ethereum 2.0 protocol. This previous document is referred to below as the "Phase 0 document".
+
+HF 1 introduces a new type of committee: the sync committee. Sync committees are responsible for signing each block of the canonical chain and there exists an efficient algorithm for light clients to sync the chain using the output of the sync committees. See the [sync protocol](./sync-protocol.md) for further details on the light client sync. Under this network upgrade, validators track their participation in this new committee type and produce the relevant signatures as required. Block proposers incorporate the (aggregated) sync committee signatures into each block they produce.
+
+## Prerequisites
+
+All terminology, constants, functions, and protocol mechanics defined in the [HF 1 -- The Beacon Chain](./beacon-chain.md) doc are requisite for this document and used throughout. Please see this document before continuing and use as a reference throughout.
+
+## Custom types
+
+| Name | SSZ equivalent | Description |
+| - | - | - |
+| `SyncCommitteeIndex` | `uint64` | Index of a validator within a sync committee |
+
+## Constants
+
+### Misc
+
+| Name | Value | Unit |
+| - | - | :-: |
+| `TARGET_AGGREGATORS_PER_SYNC_COMMITTEE` | `2**2` (= 4) | validators |
```

Comment:
could argue this should be lower, say 2, but then you have very little redundancy in what is already a small subnet...